lib/charms/traefik_k8s/v1/ingress.py

```diff
@@ -229,11 +229,11 @@ class IngressPerAppDataProvidedEvent(_IPAEvent):
     __args__ = ("name", "model", "port", "host", "strip_prefix")
 
     if typing.TYPE_CHECKING:
-        name = None  # type: Optional[str]
-        model = None  # type: Optional[str]
-        port = None  # type: Optional[str]
-        host = None  # type: Optional[str]
-        strip_prefix = False  # type: bool
+        name: Optional[str] = None
+        model: Optional[str] = None
+        port: Optional[str] = None
+        host: Optional[str] = None
```

Comment:
I thought the preferred way to do this was `name: str = None`. If that doesn't work, feel free to ignore this comment.

Reply:
It does work for **ruff**'s linting, but **pyright** complains about it!